app/controllers/api/v1/accounts/conversations_controller.rb

```diff
@@ -110,8 +110,8 @@ def set_conversation_status
   end
 
   def assign_conversation
-    @agent = Current.account.users.find(current_user.id)
-    @conversation.update_assignee(@agent)
+    @conversation.assignee = current_user
+    @conversation.save!
```

Comment:
Updating this just because I removed the update_assignee method in conversation. Also, I don't think we need to find the user again, we can use the current_user directly. @muhsin-k